From 830ea144f8b4ce74d7f828bf16f7a4e54da26fec Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:52:02 -0400
Subject: [tools] perf/stat: Fix --delay option in man page

Message-id: <1408953210-26343-235-git-send-email-jolsa@redhat.com>
Patchwork-id: 88875
O-Subject: [PATCH RHEL7.1 BZ1133083 234/322] perf stat: Fix --delay option in man page
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 8f3dd2b096c348033e55d4a4fb8c0f672559657e
Author: Andi Kleen <ak@linux.intel.com>
Date: Tue Jan 7 14:14:06 2014 -0800

description
===========
The --delay option was documented as --initial-delay in the manpage. Fix this.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/Documentation/perf-stat.txt b/tools/perf/Documentation/perf-stat.txt
index 80c7da6..29ee857 100644
--- a/tools/perf/Documentation/perf-stat.txt
+++ b/tools/perf/Documentation/perf-stat.txt
@@ -133,7 +133,7 @@ use --per-core in addition to -a. (system-wide).  The output includes the
 core number and the number of online logical processors on that physical processor.
 
 -D msecs::
---initial-delay msecs::
+--delay msecs::
 After starting the program, wait msecs before measuring. This is useful to
 filter out the startup phase of the program, which is often very different.
 
-- 
1.7.1